README-dev.md

```diff
@@ -1,105 +0,0 @@
-# Developer Documentation
-
```

Comment:
Hi @timlinux, I just realised that you removed this development environment using docker. What is your best practice for development now? Thanks